core/libraries/messages/defaults/default/email_preview_wrapper.template.php

```diff
@@ -1,4 +1,9 @@
-<?php /** @var string $main_body */ ?>
+<?php
+
+/** @var string $main_body */
+use EventEspresso\core\services\request\sanitizers\AllowedTags;
+
+?>
 <div id="ee-message-preview-container" style="margin-top: 10px">
-    <?php echo $main_body; // already escaped ?>
+    <?php echo wp_kses($main_body, AllowedTags::getWithFormTags()); ?>
 </div>
```

Comment:
ooo surprised the linter didn't scream about missing a newline at the end of the file.
maybe that only applies to classes 🤔